tensorflow/core/kernels/ctc_loss_op.cc

```diff
@@ -19,21 +19,23 @@ limitations under the License.
 #include "tensorflow/core/framework/op.h"
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/framework/register_types.h"
 #include "tensorflow/core/platform/logging.h"
 #include "tensorflow/core/platform/macros.h"
 #include "tensorflow/core/util/ctc/ctc_loss_calculator.h"
 #include "tensorflow/core/util/sparse/sparse_tensor.h"
 
 namespace tensorflow {
 
-typedef Eigen::ThreadPoolDevice CPUDevice;
+// typedef Eigen::ThreadPoolDevice CPUDevice;
```

Comment:
commented out?

Reply:
resolved in 51cd5877412f9d67f2ec180c8087ae8d7a059da1